addon/templates/components/polaris-text-field.hbs

```diff
@@ -1,13 +1,11 @@
-<!-- template-lint invalid-interactive=false -->
-
 <div class="Polaris-Labelled__LabelWrapper">
```

Comment:
Rebase onto `master` to pull in `polaris-labelled` and refactor all this, it should be much more straightforward then 😉